### PR TITLE
Minimizing repository contents

### PR DESCRIPTION
**Pull request by eddie-knight**
This minimizes any content unrelated to the content of the security baseline, and adds detailed information about how the baseline is governed.

### CODE REVIEW

File: CHARTER.md
Comment (SecurityCRob) [inline CHARTER.md:1]: We're obliged to have the charter file used across the openssf.  Can we undelete this please?

Comment (eddie-knight) [reply CHARTER.md:1]: It's currently unpopulated. Can you make a commit to the branch adding this file with contents? Or send me the file and I'll throw it in

Comment (eddie-knight) [reply CHARTER.md:1]: I've restored the original contents for now

**Review by funnelfiasco — COMMENTED** [review]
This seems good to me. The one thing we may want to consider adding is a process by which maintainers can be removed against their will (e.g. for persistent inappropriate behavior), which is never a fun thing to contemplate but it's less fun when you have to do it on the fly.

File: governance/GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+After the nomination is filed and deemed valid, [maintainer consensus] may be sought. Upon achieving consensus, the PR may be merged to confirm the new maintainer.
+
+### Sponsoring Committees
+
+To nominate a community member as a maintainer candidate, a group of maintainers may file a nomination. The committee shall meet the following criteria to be qualified to file the nomination:
+
+- The number of members in the committee shall not be less than two (2).
+- Whenever the number of organizations with maintainers in the project is more than two (2), committee members shall be from different organizations.
+
```
Comment (eddie-knight) [inline governance/GOVERNANCE.md:64]:
```suggestion
### Continued Maintainer Status

Once confirmed as a maintainer, continuation is contingent on regular activity and adherence to the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/).

```

Comment (eddie-knight) [reply governance/GOVERNANCE.md:64]: cc/ @funnelfiasco

File: README.md [hunk 2]
```diff
@@ -1,81 +1,49 @@
...
+The Open Source Project Security Baseline (OSPS Baseline) is designed to act as a minimum definition of requirements for a project relative to it's maturity level.
+
+All definitions are maintained in YAML format for tandem machine and human readibility.
+
+## Baseline Structure
+
+Each entry has the following values:
+
+- **ID**:
+  - Entries 1-39 are resurved for maturity level 1
```
Comment (cpanato) [inline README.md:12]:
```suggestion
  - Entries 1-39 are reserved for maturity level 1
```

**Review by cpanato — APPROVED** [review]
lgtm

**Review by SecurityCRob — APPROVED** [review]
+1

**Review by funnelfiasco — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,81 +1,49 @@
...
+The Open Source Project Security Baseline (OSPS Baseline) is designed to act as a minimum definition of requirements for a project relative to it's maturity level.
+
+All definitions are maintained in YAML format for tandem machine and human readibility.
+
+## Baseline Structure
+
+Each entry has the following values:
+
+- **ID**:
+  - Entries 1-39 are reserved for maturity level 1
```
Comment (david-a-wheeler) [inline README.md:12]: Approving since this is current state, but the ID ranges need to be removed.

Comment (SecurityCRob) [reply README.md:12]: objective noted, thank you

**Pull request merged**